src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/SecretUtils.java

```diff
@@ -107,6 +113,15 @@ public static String getCredentialId(Secret s) {
         return s.getMetadata().getName();
     }
 
+    public static Iterable<String> getCredentialScopes(Secret s) {
+        String scopeString = s.getMetadata().getLabels().get(SecretUtils.JENKINS_IO_CREDENTIALS_SCOPE_LABEL);
+        if (scopeString == null) {
+            return Collections.emptyList();
+        }
+
+        return Splitter.on(',').trimResults().omitEmptyStrings().split(scopeString);
```

Comment:
jobs can contain commas can't they?

Reply:
unfortunately, yes.
I could imagine to:
- Interpret the annotation as json-array of folder paths (if it starts with [ ), otherwise interpret the annotation value as a single folder path (not very elegant)
- Use another delimiter which is considered url unsafe and therefore forbidden as jenkins job name (e.g. the semicolon ; ) (not very elegant either)
- Add a special comma escaping. Since \ is url-unsafe and hence does not appear in job names,  `\,`  could be used to encode a "comma" in the job name while the plain comma could act as separator. Given that a comma will be used not that often, this sounds okaish for me - it just needs a good documentation (given that backslash might need to be double escaped in json or yaml strings)

WDYT?